docs/user-guide/configuring.md

```diff
@@ -18,7 +18,9 @@ All of these options give you fine-grained control over how ESLint treats your c
 ESLint allows you to specify the JavaScript language options you want to support. By default, ESLint expects ECMAScript 5 syntax. You can override that setting to enable support for other ECMAScript versions as well as JSX by using parser options.
 
 Please note that supporting JSX syntax is not the same as supporting React. React applies specific semantics to JSX syntax that ESLint doesn't recognize. We recommend using [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react) if you are using React and want React semantics.
-
+By the same token, supporting ES6 syntax is not the same as supporting new ES6 globals (eg new types such as `Set`.
+For ES6 syntax use `"{ parserOptions": { "ecmaVersion": 6 } }`, for new ES6 global variables use `{ "env":
+{"es6": true } }`.
```

Comment:
It probably makes sense to clarify that `env: "es6"` will both add new global variables and enable es6 syntax.